extensions/amp-riddle-quiz/0.1/test/test-amp-riddle-quiz.js

```diff
@@ -32,7 +32,8 @@ describes.realWin(
       win.document.body.appendChild(element);
     });
 
-    it('should have iframe when built', () => {
+    // TODO(#): disabled in #35227
+    it.skip('should have iframe when built', () => {
```

Comment:
It would be nice to file a ticket and associate these `TODO`s with it.

Reply:
It's on the PR's TODOs :)